lib/App.js

```diff
@@ -267,33 +142,41 @@ module.exports = class App {
 			return;
 		}
 
-		this.resetCurrent();
-
-		this.gameState = GameStateEnum.STARTING;
-		this.currentChannel = message.channel;
+		this.game = new Game(
+			this.translation,
+			this.gameMode,
+			message.channel,
+			this.players,
+		);
+		this.game.on('stop', this.resetGame.bind(this));
```

Comment:
this.game.onStop(this.resetGame.bind(this));

Reply:
There is no real benefit. Keep it simple